repl/format.go

```diff
@@ -13,12 +13,15 @@ import (
 // The value must currently be a string, list, map, and any nested values
 // with those same types.
 func FormatResult(value interface{}) (string, error) {
-	return formatResult(value)
+	return formatResult(value, false)
 }
 
-func formatResult(value interface{}) (string, error) {
+func formatResult(value interface{}, nested bool) (string, error) {
 	switch output := value.(type) {
 	case string:
+		if nested {
+			return fmt.Sprintf("%q", output), nil
```

Comment:
I haven't walked through the uses of this yet, but would it be valid to just quote all strings?

Reply:
I gather that this is currently used by folks to extract string values directly without further parsing, so to quote them here would be a breaking change. We might still make that change for 0.12, since this'll get rewritten then anyway.